pkg/config/config.go

```diff
@@ -249,6 +272,9 @@ fs:
 couchdb:
     url: http://localhost:5984/
 
+cache:
+    url: redis://localhost:6379
```

Comment:
Can we make it explicit that we use the database 0, like this: `redis://localhost:6379/0`?